lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -21,6 +21,7 @@ sub new {
 
 sub expect_is_shown {
     my ($self) = @_;
+    send_key_until_needlematch($self->{tag_first_entry_highlighted}, 'down');
```

Comment:
I would rename that to `tag_agama_installer_highlighted`, otherweise it is confusing because it is not neccesarily the "first" entry.

Reply:
Updated.